File: Ent_UsAdmin/perfilalt.vb
Public Class perfilalt

    
    Private Sub BunifuThinButton21_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles BunifuThinButton21.Click
        editarperfil.Show()
        frm_adm.abierto = False
        frm_adm.Timer2.Stop()

        Me.Close()

    End Sub

    Private Sub perfilalt_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        If imgstrusr = "" Then
            PictureBox2.Image = ConfiaAdmin.My.Resources.Resources.usuarionegro
        Else
            PictureBox2.Image = bitmapimgusr
        End If
        If nm_completeusr <> "" Then
            lblnm_complete.Text = nm_completeusr
        End If

        If addrusr <> "" Then
            lbldireccion.Text = addrusr
        End If

        If tlfusr <> "" Then
            lbltlf.Text = tlfusr
        End If

        If grpusr <> "" Then
            lblgrp.Text = grpusr
        End If

        If nmusr <> "" Then
            lblnm.Text = nmusr
        End If

        If idSesion <> "" Then
            lblSesion.Text = idSesion
        End If

    End Sub

    Private Sub BunifuThinButton22_Click(sender As Object, e As EventArgs) Handles BunifuThinButton22.Click
        frm_adm.cerrarSesion = True
        frm_adm.Close()
    End Sub
End Class

File: Ent_UsAdmin/perfilalt.cs
using System;

namespace ConfiaAdmin
{
    public partial class perfilalt
    {
        public perfilalt()
        {
            InitializeComponent();
        }


        private void BunifuThinButton21_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.editarperfil.Show();
            My.MyProject.Forms.frm_adm.abierto = false;
            My.MyProject.Forms.frm_adm.Timer2.Stop();

            Close();

        }

        private void perfilalt_Load(object sender, EventArgs e)
        {
            if (string.IsNullOrEmpty(Module1.imgstrusr))
            {
                PictureBox2.Image = My.Resources.Resources.usuarionegro;
            }
            else
            {
                PictureBox2.Image = Module1.bitmapimgusr;
            }
            if (!string.IsNullOrEmpty(Module1.nm_completeusr))
            {
                lblnm_complete.Text = Module1.nm_completeusr;
            }

            if (!string.IsNullOrEmpty(Module1.addrusr))
            {
                lbldireccion.Text = Module1.addrusr;
            }

            if (!string.IsNullOrEmpty(Module1.tlfusr))
            {
                lbltlf.Text = Module1.tlfusr;
            }

            if (!string.IsNullOrEmpty(Module1.grpusr))
            {
                lblgrp.Text = Module1.grpusr;
            }

            if (!string.IsNullOrEmpty(Module1.nmusr))
            {
                lblnm.Text = Module1.nmusr;
            }

            if (!string.IsNullOrEmpty(Module1.idSesion))
            {
                lblSesion.Text = Module1.idSesion;
            }

        }

        private void BunifuThinButton22_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.frm_adm.cerrarSesion = true;
            My.MyProject.Forms.frm_adm.cerrarEmpresa = false;
            My.MyProject.Forms.frm_adm.Close();
        }

        private void BunifuThinButton23_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.frm_adm.cerrarSesion = true;
            My.MyProject.Forms.frm_adm.cerrarEmpresa = true;
            My.MyProject.Forms.frm_adm.Close();
        }
    }
}